src/librustc_typeck/check/wfcheck.rs

```diff
@@ -711,15 +716,9 @@ impl<'a, 'gcx, 'tcx> FnCtxt<'a, 'gcx, 'tcx> {
     }
 }
 
-fn error_192(tcx: TyCtxt, span: Span) {
-    span_err!(tcx.sess, span, E0192,
-              "negative impls are only allowed for traits with \
-               default impls (e.g., `Send` and `Sync`)")
-}
-
 fn error_380(tcx: TyCtxt, span: Span) {
     span_err!(tcx.sess, span, E0380,
-              "traits with default impls (`e.g. impl \
+              "traits with auto impls (`e.g. impl \
```

Comment:
aren't these just called `auto trait` now or something? I thought we landed some such PR.

Reply:
We added `auto trait` (https://github.com/rust-lang/rust/pull/45247), but not removed `impl Trait for ..` yet (https://github.com/rust-lang/rust/pull/46480).